lib/bot/issue_manager.rb

```diff
@@ -144,6 +144,20 @@ def add_labels(command_value, author, issue)
     end
     if !valid_labels.empty?
       issue.add_labels(valid_labels)
+      notify_on_label(issue, valid_labels)
+    end
+  end
+
+  def notify_on_label(issue, labels)
+    labels.each do |label|
+      if Settings.label_notifier.contacts[label] &&
+         Settings.label_notifier.contacts[label] > 0
+        message = 'cc '
+        Settings.label_notifier.contacts[label].each do |contact|
+          message += "@#{contact}"
+        end
+        issue.add_comment(message)
```

Comment:
This is going to create a separate comment per label in addition to the initial comment response.  Can we somehow tack it into the regular response so we don't flood the issue?